### DISCUSSION TITLE
Color mode based on device settings

### DISCUSSION THREAD

**Discussion by haibert** [GeekyAnts/NativeBase#4361] [Q&A] [answered]
Hey guys, 
is there a way to have the color mode be automatically switched when device settings are changed from light to dark? Instead of using toggleColorMode?

**Comment by kaili-kameoka** [comment, marked as answer]
You can create a custom theme with a config like this:
```javaScriptconst config = {
    useSystemColorMode: true
  };

  const customTheme = extendTheme({ config });
```

Then, pass the custom theme in the theme prop of the `NativeBaseProvider`

Docs: https://docs.nativebase.io/color-mode

If you're using expo, you may also need to modify the app.json so that the ios and android platforms have the following: 
```JSON
"userInterfaceStyle": "automatic",
```
Docs: https://docs.expo.dev/guides/color-schemes/
